providers/giphy/giphy.go

```diff
@@ -0,0 +1,25 @@
+package giphy
+
+import (
+	"fmt"
+	"net/url"
+
+	"github.com/zquestz/s/providers"
+)
+
+func init() {
+	providers.AddProvider("giphy", &Provider{})
+}
+
+// Provider merely implements the Provider interface.
+type Provider struct{}
+
+// BuildURI generates a search URL for Giphy.
+func (p *Provider) BuildURI(q string) string {
+	return fmt.Sprintf("https://giphy.com/search/%s", url.QueryEscape(q))
+}
+
+// Tags returns the tags relevant to this provider.
+func (p *Provider) Tags() []string {
+	return []string{"gifs"}
```

Comment:
This should be 'pics'. We don't need a brand new tag for just gifs.

Reply:
oh i see, no prob  👌🏻